src/domains/aca-app/api/gpd-for-aca-services/v1/_base_policy.xml

```diff
@@ -0,0 +1,81 @@
+<policies>
+  <inbound>
+    <choose>
+      <when condition="@(Regex.IsMatch(context.Request.Url.Path, @"^\/pagopa-gpd-core\/organizations\/([a-zA-Z0-9]{11})\/debtpositions$", RegexOptions.IgnoreCase, TimeSpan.FromMilliseconds(500)) && ((string)(context.Request.Method)).Equals("POST"))">
```

Comment:
move from all-api to single api (for example https://github.com/pagopa/pagopa-infra/blob/f6d39b2a9e336725cf871a43f6a9371b50e82463/src/core/apim_payment_manager.tf#L930)